app/views/user_mailer/claim_notification.html.erb

```diff
@@ -1,17 +1,17 @@
 <% content_for :message do %>
   <h2 class="heading">
-    Hello from the Archive of Our Own!
+    <%= t '.hello' %>
   </h2>
 
-  <p>You're receiving this e-mail because you had works in a fanworks archive that has been imported by <%= style_link('Open Doors', 'http://opendoors.transformativeworks.org/') %> into the <%= style_link('Archive of Our Own', root_url) %> (AO3). Because this e-mail address is connected to one registered on the imported archive, the associated fanworks (listed below) have been automatically added to your AO3 account.</p>
+  <p><%= t( '.html.part1', open_doors: style_link(t('.open_doors'),'http://opendoors.transformativeworks.org'), ao3: style_link(t('.ao3'), root_url )).html_safe %></p>
```

Comment:
I think we have some unnecessary spaces:
* `t( '.html.part1',`
* `root_url )).html_safe`

And one space missing after a comment:
* `('.open_doors'),'http://opendoors.transformativeworks.org')`